scripts/comicdb.js

```diff
@@ -66,4 +66,40 @@ jQuery(document).ready(function($) {
       $('.form-add-list').addClass('active');
     }
   }
+
+  // Sort display controls
+  var $sortControls, $comics, $comicsList;
+
+  $sortControls = $('.sort-control');
+  $comicsList = $('#inventory-table');
+  $comics = $($comicsList).find('li');
+
+  $($sortControls).click(function() {
+    var controlId, layoutThumbLg, layoutThumbSm, layoutList;
+    controlId = $(this).attr('id');
```

Comment:
`$(this).attr('id')` is grabbing the ID of the button being pressed so it can be passed to the loop below.